Dockerfile

```diff
@@ -39,4 +39,5 @@ RUN yarn workspace @grain/compiler esy compile
 RUN yarn workspace @grain/compiler esy copy-compiler
 
 # Set up container environment
-WORKDIR /
+WORKDIR /grain
```

Comment:
Why do we want the working dir to be the grain project itself if we want the default command to be executing grain?